source/GameplayChangersSubstate.hx

```diff
@@ -91,7 +91,7 @@ class GameplayChangersSubstate extends MusicBeatSubstate
 		var option:GameplayOption = new GameplayOption('Botplay', 'botplay', 'bool', false);
 		optionsArray.push(option);
 
-		var option:GameplayOption = new GameplayOption('Play Opponents Chart', 'opponentplay', 'bool', false);
+		var option:GameplayOption = new GameplayOption("Play Opponent's Chart", 'opponentplay', 'bool', false);
```

Comment:
I think a better name for this could be "Play as Opponent"

Reply:
shish kebab